### PR TITLE
Added and fixed Tulane faculty.

### PR DESCRIPTION
**Pull request by nmattei**
Added Nicholas Mattei at Tulane and fixed DBLP name for K. Brent Venable.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -7040,7 +7040,7 @@ Jürgen Giesl,RWTH Aachen,http://verify.rwth-aachen.de/giesl,dh9YNEoAAAAJ
 Jürgen Sachau,University of Luxembourg,http://wwwen.uni.lu/snt/people/juergen_sachau,NOSCHOLARPAGE
 Jürgen Schmidhuber,Università della Svizzera italiana,http://search.usi.ch/people/855dfcba3eaf6e94156db5ff991ba300/schmidhuber-juergen,gLnCTgIAAAAJ
 K. Anton Feenstra,VU Amsterdam,http://www.few.vu.nl/~feenstra,NOSCHOLARPAGE
-K. Brent Venable,Tulane University,http://www2.tulane.edu/sse/cs/faculty/k-brent-venable.cfm,iqkTonoAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Both should be in here, along with an alias link.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (nmattei) [reply csrankings.csv]: Not sure I understand -- should I list her in the main file as K. and in the alias as Kristen?

Comment (nmattei) [reply csrankings.csv]: So in the dblp alises it has her as Kristen which matches her DBLP name (https://dblp.uni-trier.de/pers/hd/v/Venable:Kristen_Brent) but it is not pulling her record correctly for the ranking...  I can't quite figure out how to fix it...

Comment (emeryberger) [reply csrankings.csv]: list both versions in the main file, and add an entry to the aliases file.

Comment (nmattei) [reply csrankings.csv]: Okay, I believe that commit did it -- sorry for not renaming the branch, was in a rush yesterday.

**Comment by nmattei** [comment]
I have implemented the requested changes.

**Pull request merged**